app/src/utils/utils.ts

```diff
@@ -415,7 +415,7 @@ export const shuffle = (unshuffled: Grant[] | GrantRound[]) => {
 };
 
 /**
- * @notice Recursively grab every page of results
+ * @notice Recursively grab every 'page' of results - please note that the query MUST return the ID field
```

Comment:
Given that `subgraphUrl`s in `chains.ts` are currently `false`, is there a way to test out this PR in the dgrants app? I don't think so, so perhaps we convert this to draft, and once the new subgraph is deployed we add the subgraph URLs in this same PR and test?

Reply:
Yes, that sounds good to me! As we discussed yesterday, I tested this against the old graph URLs but ran in to some unrelated errors so wasn't fully able to verify that everything is sound. I think setting this as a draft is a good call - I'll get that done now 👍

Reply:
The subgraphs are now deployed so I'm opening this back up for review 🙌